legal-api/src/legal_api/services/filings/validations/restoration.py

```diff
@@ -102,15 +113,52 @@ def validate_party(filing: Dict) -> list:
     return msg
 
 
-def validate_restoration_court_order(filing: dict, restoration_type: str) -> list:
+def validate_approval_type(filing: dict, restoration_type: str, limited_restoration: Filing) -> list:
+    """Validate approval type."""
+    msg = []
+    approval_type = get_str(filing, APPROVAL_TYPE_PATH)
+
+    if not approval_type:
+        msg.append({'error': 'Must provide approval type.', 'path': APPROVAL_TYPE_PATH})
```

Comment:
`approvalType` is a required and not nullable field in schema. Do we need this `if` here?

Reply:
good call, will update

Reply:
removed